### ISSUE TITLE
Line Height not set anymore ?

### ISSUE THREAD

**Issue by Creat1veM1nd** [5axes/UniversalCuraSettings#35]
Hello, 

I noticed that with Cura 4.x the line height is set, e.g. to 0.24 in mode prototyope. But in Cura 5.x it is not set anymore (and obviously some other values too, because in Cura 4.x 99 values were set, in Cura 5.x 36). 

Is that on purpose or is the addon for Cura 5 still work in progress ?

Thanks for a hint!

**Comment by 5axes** [comment]
In the last release the option "set standard settings" must be checked if you want also to modify this value. This option was added to reduce the number of modified parameter if you just want the modification of dedicate parameter and not a complete profile

**Comment by Creat1veM1nd** [comment]
I tried with checked and unchecked - but line height is not set. I am using version 0.16 (according to Cura Marketplace)

**Comment by Creat1veM1nd** [comment]
I looked into it a bit more and tried if any other plugin prevents from setting. But none does. While restarting Cura quite a few times and testing the settings, I noticed that the very first change of settings via Universal Cura Settings works as expected. But every other turn, even with change printer profile (I do have two printers in Cura) does not work properly anymore. When restarting Cura again, I can again set all values exactly one time.

**Comment by 5axes** [comment]
Yes, this is a normal behaviour, once the basic settings are set normally they are not changed. But maybe the operation needs to be reviewed.

**Comment by Creat1veM1nd** [comment]
I just tested it and I was right: 

I do start Cura, choose prototype from UCS -> all set. After that I choose small part from UCS -> only some set, but not e.g. line height.

**Comment by Creat1veM1nd** [comment]
Do you need any further info or tests ? I would be glad to help out.

**Comment by 5axes** [comment]
Actual release on this repositery 0.0.10, the cura marketplace is not up to date. For me it works as intended. But if if you have any comment or bug report or any feedback about your own results, they are welcome.

**Comment by Creat1veM1nd** [comment]
I just installed the Curapackage with release 0.1.10 and it seems to work. I will test it some more (might take some time, quite busy right now). So far it looks good.
